### DISCUSSION TITLE
Why are there so many vlog files under my node? Can I delete them？

### DISCUSSION THREAD

**Discussion by crow123** [filecoin-project/lotus#6136] [Storage Provider] [answered]
![image](https://user-images.githubusercontent.com/15045264/116237239-8eeea380-a792-11eb-9f73-87349d3dda1a.png)

**Comment by magik6k** [comment, marked as answer]
What directory are they in?

If this is in `.lotus/datastore/chain`, this is chain data, and removing them will break your node.

There is a way to reduce the size of this directory - follow the steps in https://docs.filecoin.io/get-started/lotus/chain/#compacting-the-chain-data

(If this isn't `.lotus/datastore/chain`, then it depends on exactly what directory you're talking about)
